### PR TITLE
Revert #2074 and remove closed Milliman internship

### PR DESCRIPTION
**Pull request by tianyizheng02**
Revert #2074 because
- Internship wasn't for the summer term
- Link was from LinkedIn, not the company's own career site
- Location was incorrect

Moved the link to the off-season README with the correct info.

Removed closed Milliman summer 2024 internship.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
...
-| [Milliman](https://recruiting2.ultipro.com/MIL1017/JobBoard/f54234e9-dfde-b183-fd20-4fbdb19cba7a/OpportunityDetail?opportunityId=07e563a5-18f6-4e9f-a719-452971adba46)| Omaha, NE <br/>  Windsor, CT | Software Engineer Intern <br/> [Healthcare Analytics Intern Spring 2024 (CMH Health)](https://www.linkedin.com/jobs/view/3666135380/?eBP=JYMBII_JOBS_HOME_ORGANIC&recommendedFlavor=ACTIVELY_HIRING_COMPANY&refId=rnKyo6MZN%2BDDkzq3b2Jziw%3D%3D&trackingId=x%2FyKzqB58ORjCaZUDKFijQ%3D%3D&trk=flagship3_jobs_discovery_jymbii)|
+| [Milliman](https://recruiting2.ultipro.com/MIL1017/JobBoard/f54234e9-dfde-b183-fd20-4fbdb19cba7a/OpportunityDetail?opportunityId=07e563a5-18f6-4e9f-a719-452971adba46)| Omaha, NE | Software Engineer Intern |
```
Comment (ncbarta) [inline README.md]: I'm getting "Sorry, this opportunity is only available for Milliman, Inc. employees." from this link. 

Their [job board](https://recruiting2.ultipro.com/MIL1017/JobBoard/f54234e9-dfde-b183-fd20-4fbdb19cba7a/?q=&o=postedDateDesc&f5=pzIognlofkSxWxZlEgjctg) does not show any summer internship postings at this time.

I have verified that the job board is reachable from their website careers page.

File: README-Off-Season.md [hunk 2]
```diff
@@ -67,10 +67,11 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
...
 | [Medpace](https://uscareers-medpace.icims.com/jobs/8878/software-development-co-op---fall-2023/job) | Cincinnati, OH | Fall 2023 | Software Development Intern |
-| [Skydio](https://boards.greenhouse.io/skydio/jobs/5639641003?gh_src=b2366f993us) | San Mateo, California | Fall 2023 | Embedded Software Engineer Intern |
+| [Skydio](https://boards.greenhouse.io/skydio/jobs/5639641003?gh_src=b2366f993us) | San Mateo, CA | Fall 2023 | Embedded Software Engineer Intern |
 | Impulse Space | Redondo Beach, CA | Fall 2023 | **🔒 Closed 🔒** Embedded Software Engineering Intern (US Citizenship Required) |
 | BenefitMall | Hybrid (Dallas, TX) | Multiple | **🔒 Closed 🔒** Software Development Intern |
 | [Tesla](https://www.tesla.com/careers/search/?site=US&type=3) | Multiple | Fall 2023, Winter/Spring 2024 | Frontend/Backend SWE, Network Engineering, PM, UI/UX, and more. |
+| [Milliman](https://recruiting2.ultipro.com/MIL1017/JobBoard/f54234e9-dfde-b183-fd20-4fbdb19cba7a/OpportunityDetail?opportunityId=13902acc-1b58-407a-8c0f-c093d7b8cc5a) | Milwaukee, WI <br/> Chicago, IL <br/> Hartford, CT <br/> Remote | Spring 2024 | Healthcare Analytics Intern |
```
Comment (ncbarta) [inline README-Off-Season.md:74]: Looks good

File: README.md (hunk 1 above)
Comment (tianyizheng02) [reply README.md]: Will mark it as closed then

Comment (ncbarta) [reply README.md]: Are we sure it was ever open?

**Pull request merged**